test/README.md

```diff
@@ -83,8 +83,27 @@ Makefile targets
     test_results/compilable/json.d.out      runs an individual test
                                             (run log of the test is stored)
 
-In-test variables
+Test Configuration
 -----------------
```

Comment:
As mentioned I think some Markdown engines don't like this and require a newline after the heading.